lib/StringClient.js

```diff
@@ -183,10 +200,13 @@ StringClient.prototype.write = function write(options, body, callback) {
 
 
 StringClient.prototype.parse = function parse(req, callback) {
+    var self = this;
+
     function parseResponse(err, res) {
         var body = '';
         var gz;
         var hash;
```

Comment:
Can you rename these variables so they're more clear? Perhaps `default_ecoded_hash` and `latin_encoded_hash`

Reply:
Will do.